src/pages/nonprofit/issues/create.tsx

```diff
@@ -1,9 +1,225 @@
+import { Button } from "@chakra-ui/button";
+import { Heading, Stack, Text, VStack, Link } from "@chakra-ui/layout";
+import { Radio, RadioGroup } from "@chakra-ui/radio";
 import { Flex } from "@chakra-ui/react";
+import { Textarea } from "@chakra-ui/textarea";
+import { useEffect, useState } from "react";
+import { HiOutlineChevronLeft } from "react-icons/hi";
+import { nonprofitCreateIssue } from "src/actions/Issue";
+import { nonprofitGetProject } from "src/actions/Project";
+import { showError, showInfo } from "src/utils/notifications";
+import {
+  Chapter,
+  IssueStatus,
+  MaintenanceType,
+  NonprofitCreateIssue,
+  Project,
+  ProjectStage,
+  UploadedFile,
+} from "src/utils/types";
+import { uploadFile } from "src/utils/uploaded-files";
+import urls from "src/utils/urls";
 
 function NonprofitIssueCreationPage() {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [issueType, setIssueType] = useState<MaintenanceType>();
+  const [project, setProject] = useState<Project>();
+  const [preUploadImages, setPreUploadImages] = useState<FileList>();
+  const [images, setImages] = useState<UploadedFile[]>([]);
+  const [chapterIssueTypes, setChapterIssueTypes] = useState<MaintenanceType[]>(
+    [MaintenanceType.BUG_FIXES, MaintenanceType.NEW_FEATURES]
+  );
+  const [projectId, setProjectId] = useState("617ea71c069ce109e2ae9f83");
+
+  useEffect(() => {
+    async function getProject() {
+      let project;
+      try {
```

Comment:
This try catch block is unnecessary since you are already catching the error when calling this function.  That will also make it so you do not have to declare the project variable before assigning a value, so you can make it a constant.